packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.tsx

```diff
@@ -55,6 +55,7 @@ type QueryWrapperPlotNavOwnProps = {
   /** Whether the displayed results should be paginated or infinite. Default for cards is true, default for table is false */
   isInfinite?: boolean
   sql: string
+  showAccessColumnHeader?: boolean
```

Comment:
Remove this new prop and just use the existing `tableConfiguration` prop (you already added it to the `SynapseTableConfiguration` type).